test/integration/targets/ec2_eip/tasks/main.yml

```diff
@@ -43,7 +43,7 @@
         state: present
         in_vpc: yes
         <<: *aws_connection_info
-        public_ip: "{{ eip.public_ip }}"
+        public_ip: "{{ eip.public_ip[0] }}"
```

Comment:
This would be a breaking change.  ec2_eip is flagged as `stableinterface` we shouldn't be breaking this.

Reply:
Indeed, refer to my below comment.